components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -436,12 +436,25 @@ class MozillaSocorroService(
                         "Content-Type: application/octet-stream\r\n\r\n"
                     ).toByteArray()
             )
+        } catch (e: IOException) {
+            logger.error("failed to write boundary", e)
+            return
+        }
+
+        try {
             val fileInputStream = FileInputStream(file).channel
             fileInputStream.transferTo(0, fileInputStream.size(), Channels.newChannel(os))
             fileInputStream.close()
         } catch (e: IOException) {
             logger.error("failed to send file", e)
         }
+
+        try {
+            // Add EOL in case minidump is empty
```

Comment:
This is less in case the minidump is empty and more that you always have to have a `\r\n` separating the content from the next part header.

Reply:
Sure, I'll update the comment.  Thanks!